src/home/components/RecentCard.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { StyleSheet, SafeAreaView, Text, ScrollView} from 'react-native';
+
+const RecentCard = ({ navigation }) => {
+  return (
+    <SafeAreaView style={{alignItems: 'center', justifyContent: 'center'}}>
+
+        <SafeAreaView style={styles.box}>
+
+          <ScrollView>
+            {/* <Text>CARD GOES HERE</Text> */}
+          </ScrollView>
+
+        </SafeAreaView>
+
+    </SafeAreaView>
+  );
+};
+
+const styles = StyleSheet.create({
+  box: {
+    backgroundColor: 'white',
+    borderWidth: 1,
+    borderColor: 'black',
+    borderRadius: 20,
+    alignItems: 'center',
+    justifyContent: 'center',
+    overflow: 'hidden',
+    marginVertical: 15,
+    width: 332,
+    height: 187
```

Comment:
Are these values responsive? The card should expand in width and height according to different screen sizes. You can test by wrapping a View box around it, and adjusting the View's width and height to see what happens.